.github/workflows/ci.yml

```diff
@@ -11,6 +11,12 @@ on:
 permissions:
   contents: read
 
+env:
+  LUA_VERSION: "5.1"
```

Comment:
Maybe we should use matrix here instead of vars, will allow to easily extend to other versions and test them all.

Reply:
Thank you!

Reply:
Done.

Thrashed around for a while trying to use environment variables for matrix values but failed.

The matricies do work, setting `nvim_version: [ v0.8.3, v0.9.4 ]` worked but failed as the annotations are not present in 0.8.3.

Reply:
I'd be most grateful if you could test the makefile as per `CONTRIBUTING.md`